Sources/WalletConnectNotify/Client/Wallet/SubscriptionWatcher.swift

```diff
@@ -26,26 +26,23 @@ class SubscriptionWatcher {
         self.notificationCenter = notificationCenter
     }
 
-    func setupTimer() {
-        onSetupTimer?()
-        logger.debug("Setting up Subscription Watcher timer")
-        timerCancellable?.cancel()
-        timerCancellable = Timer.publish(every: timerInterval, on: .main, in: .common)
-            .autoconnect()
-            .sink { [weak self] _ in
-                self?.backgroundQueue.async {
-                    self?.watchSubscriptions()
-                }
-            }
-    }
+    deinit { stop() }
```

Comment:
Do we need this?
client is a singleton and will never be deinitialised

Reply:
It's for unit tests